app/controllers/accesses_controller.rb

```diff
@@ -64,6 +66,10 @@ def set_access
       @access = Access.find(params[:id])
     end
 
+    def get_accesses_collection
```

Comment:
Consider moving this method inside ApplicationController along with other helper methods.

Reply:
@anantaUM  get_accesses_collection method is a private method for the AccessesController. I think It should not be available to other controllers.

Reply:
@britaumich - ApplicationController has all helper methods related to Role / Permission / Access. Hence, I think it is better to move it there to avoid repetition in case we want to use same logic elsewhere in future.

Reply:
@anantaUM ApplictionController has methods that are available to the whole application. If a method is called only in one controller it should stay in that controller. 
get_accesses_collection is called only once, and it should not be a method at all.  
Moving part of this statement 
`@accesses = Access.where(permission_id: current_user_permissions).order(:uniqname)
` 
to a method complicates things and adds confusion.

Reply:
moved to Application Controller